src/login/logind-dbus.c

```diff
@@ -1905,6 +1910,7 @@ static int method_do_shutdown_or_sleep(
 
 static int method_poweroff(sd_bus_message *message, void *userdata, sd_bus_error *error) {
         Manager *m = userdata;
+        bool root_enforce_inhibit = sd_bus_message_is_method_call(message, NULL, "PowerOffWithInhibitors");
```

Comment:
nit: we don't mix function calls with declarations, split these lines. See: docs/CODING_STYLE.md